### DISCUSSION TITLE
new ,broadcasts, people

### DISCUSSION THREAD

**Discussion by PandaWhoCodes** [tweepy/tweepy#1153] [Questions] [answered]
How to get news,broadcast, people and photos using tweepy ?
for example to get news on machine learning 
I use the following url
https://twitter.com/search?f=news&vertical=default&q=%22machine%20learning%20platform%22&src=typd

where f=news

**Comment by Harmon758** [comment, marked as answer]
Twitter's [standard search API](https://developer.twitter.com/en/docs/tweets/search/api-reference/get-search-tweets.html) doesn't support filters directly.
However, you should be able to just use [the standard filter operator](https://developer.twitter.com/en/docs/tweets/rules-and-filtering/overview/standard-operators.html) in your search query.
e.g. `"machine learning platform" filter:news`

**Comment by PandaWhoCodes** [comment]
Thanks
